Automattic/wp-calypso
# Post Editor: Add stats for the Gutenberg opt-in dialog.

This PR adds stats for the buttons in the opt-in dialog.

<img width="1100" alt="screen shot 2018-09-11 at 4 32 06 pm" src="https://user-images.githubusercontent.com/349751/45393345-58af8600-b5e0-11e8-94e1-b1c912732fb0.png">

**Testing**

* Observe stats activity in the console with `localStorage.setItem( 'debug', 'calypso:analytics*' );`
* Open the post editor to see the opt-in visual at the bottom of the sidebar.
* Clicking the dialog's `X` button and clicking outside of the dialog close the dialog, with no stats recorded.
* Click both the "Try..." and "stick to classic" buttons.
* "Try...": closes the dialog and forwards the user to a `/gutenberg` route.
* "classic" just closes the dialog.
* Both buttons send the appropriate stats, eg:

<img width="983" alt="screen shot 2018-09-13 at 2 41 59 pm" src="https://user-images.githubusercontent.com/349751/45517622-9be63200-b763-11e8-8ce5-38f49e032c47.png">
